src/worker/core/draft/genPlayersWithoutSaving.ts

```diff
@@ -57,12 +58,15 @@ const genPlayersWithoutSaving = async (
 	// Do one season at a time, keeping the lowest pot players in college for another season
 	let enteringDraft: typeof remaining = [];
 
-	for (let i = 0; i < 4; i++) {
```

Comment:
Somehow the diff is wrong... or it somehow has not updated since I made changes to this file for hockey. Wonder if that's happening elsewhere too? If the correct diff was showing, you'd see that your version of this file lost the changes I made for hockey.

Reply:
I haven't updated the branch yet so this file is outdated to before hockey was made